eZ/Publish/Core/Persistence/Cache/UrlAliasHandler.php

```diff
@@ -251,26 +257,33 @@ public function locationMoved($locationId, $oldParentId, $newParentId)
         );
 
         $return = $this->persistenceHandler->urlAliasHandler()->locationMoved($locationId, $oldParentId, $newParentId);
-
         $this->cache->clear('urlAlias');//TIMBER! (Will have to load url aliases for location to be able to clear specific entries)
+
         return $return;
     }
 
     /**
      * @see eZ\Publish\SPI\Persistence\Content\UrlAlias\Handler::locationCopied
      */
-    public function locationCopied($locationId, $oldParentId, $newParentId)
+    public function locationCopied($locationId, $newLocationId, $newParentId)
     {
         $this->logger->logCall(
             __METHOD__,
             array(
-                'location' => $locationId,
-                'oldParent' => $oldParentId,
+                'oldLocation' => $locationId,
+                'newLocation' => $newLocationId,
                 'newParent' => $newParentId,
             )
         );
 
-        return $this->persistenceHandler->urlAliasHandler()->locationCopied($locationId, $oldParentId, $newParentId);
+        $return = $this->persistenceHandler->urlAliasHandler()->locationCopied(
+            $locationId,
+            $newLocationId,
+            $newParentId
+        );
+        $this->cache->clear('urlAlias');//TIMBER! (Will have to load url aliases for location to be able to clear specific entries)
```

Comment:
I think this is not a solution for when you will hit the UrlAliasHandler->loadUrlAlias($aliasId) this will return also not accured information because the id is also stored and we need to clean it up.